assets/javascripts/discourse/templates/group/assigned.hbs

```diff
@@ -12,9 +12,23 @@
       </div>
     {{/if}}
     {{#load-more selector=".activity-nav li" action=(action "loadMore")}}
-      {{group-assigned-filter show-avatar=false filter="everyone" routeType=route_type assignmentCount=group.assignment_count}}
+      {{group-assigned-filter
+        show-avatar=false
+        filter="everyone"
+        routeType=route_type
+        assignment_count=group.assignment_count
+        q=q
+        ascending=ascending
+        order=order}}
       {{#each members as |member|}}
-        {{group-assigned-filter show-avatar=true filter=member routeType=route_type}}
+        {{group-assigned-filter
+          show-avatar=true
+          filter=member
+          routeType=route_type
+          assignment_count=group.assignment_count
```

Comment:
Do we need to pass the group assignment count here? I think we only need it for the 'everyone' button?